examples/pubsub_realtime/README_ETF.txt

```diff
@@ -1,92 +0,0 @@
-PRE-REQUISITES:
```

Comment:
Why are are you removing this file?
I understand that ETF and XDP are currently independent.
One for publishing, one for subscribing.

Reply:
Updated the existing README_ETF.txt file and changed the file name